code/modules/admin/verbs/lua/README.md

```diff
@@ -121,7 +121,7 @@ Yields the current thread, scheduling it to be resumed during the next fire of S
 ---
 
 ## The SS13 package
-The `SS13` package contains various helper functions that use code specific to tgstation.
```

Comment:
What I said for bug_report.md's 9th, feedback_conversion_2017-11-12.py's 493rd, wrestling.dm's 3rd and this file's 93rd line applies for this line.